appium-cookies/.gitignore

```diff
@@ -0,0 +1,4 @@
+node_modules
+build
+*.log
+.DS\_Store
```

Comment:
When I tried running 
git config --global core.excludesfile ~/.gitignore_global
it gave me a permission denied error